ui/pages/confirmations/components/confirm/info/contract-interaction/contract-interaction.tsx

```diff
@@ -40,6 +47,15 @@ const ContractInteractionInfo: React.FC = () => {
       >
         <TransactionDetails />
       </Box>
+      {showAdvancedDetails && (
+        <Box
```

Comment:
Assuming the `AdvancedDetails` component will always be in confirmations, then could the surrounding `Box` container be moved into it?

Reply:
My thinking here was that, considering `SimulationDetails` needed a box wrapper, we would wrap all the components. I suppose other components are different though, in that they are specific for the contract interaction component as you point out.

This is a good suggestion, and I did the same for `TransactionDetails`.